wallet/src/main/proto/wallet.proto

```
@@ -0,0 +1,133 @@
+syntax = "proto3";
+
+package bisq.wallet.grpc;
```

Buf (1.55.1)

Files with package "bisq.wallet.grpc" must be within a directory "bisq/wallet/grpc" relative to root but were in directory "wallet/src/main/proto".

(PACKAGE_DIRECTORY_MATCH)